README.md

```diff
@@ -33,28 +33,42 @@ reStructuredText Docstring Format is the standard. Here is an example:
 
 ### How to update the documentation:
 
-To update the docs branch, checkout the gh-pages branch. Make sure it is up to
-date, then clone the dataprofiler from the feature branch you want in order to
-update the documentation (probably main).
+1. Either with an existing clone of `capitalone/dataprofiler` or clone the `capitalone/dataprofiler` reposotory to your local computer with the following command: 
+```bash
+git clone https://github.com/capitalone/DataProfiler dataprofiler
```

Comment:
Do we need to enforce downloading it with that name?

Reply:
don't have to

Reply:
Let's not enforce it. In the case someone sets up a second repo, this could cause conflicts.

Reply:
gotcha so will default to `DataProfiler` ... could be the same issue with conflicts, but most folks should have a `capitalone/dataprofiler` clone already so this `clone` command shouldn't need to happen for most folks on our team